src/main/scala/mesosphere/marathon/api/MarathonExceptionMapper.scala

```diff
@@ -73,7 +73,7 @@ class MarathonExceptionMapper extends ExceptionMapper[JavaException] {
 
   private def entity(exception: JavaException): JsValue = exception match {
     case e: NotFoundException =>
-      Json.obj("message" -> s"URI not found: ${e.getNotFoundUri.getRawPath}")
+      Json.obj("message" -> "URI not found")
```

Comment:
I think you should say which URI was not found.

Reply:
The path is not available on this exception anymore.

Reply:
Also, why do we have to send this to the client? Won't the client know which URL it requested?

Reply:
It's easier to debug in the logs. However, if the exception does not have the uri it's fine.

Reply:
Does this affect the logs? My reading is this is what we send to the client.